dowhy/gcm/stats.py

```diff
@@ -161,3 +164,65 @@ def permute_features(feature_samples: np.ndarray,
             np.random.shuffle(feature_samples[:, feature])
 
     return feature_samples
+
+
+def estimate_f_test_p_value(X_training_a: np.ndarray,
+                            X_training_b: np.ndarray,
+                            Y_training: np.ndarray,
+                            X_test_a: np.ndarray,
+                            X_test_b: np.ndarray,
+                            Y_test: np.ndarray) -> float:
+    """ Estimates the p-value for the nullhypothesis that the same regression curve with less parameters can be
```

Comment:
"same regression curve" or the "same regression error"?

Reply:
I think its kind of equivalent here, the same curve would lead to the same error. But I changed it to "same regression error" to be clearer.